internal/configs/annotations.go

```diff
@@ -93,29 +89,41 @@ func parseAnnotations(ingEx *IngressEx, baseCfgParams *ConfigParams, isPlus bool
 		if err != nil {
 			glog.Error(err)
 		}
-		cfgParams.HealthCheckEnabled = healthCheckEnabled
+		if isPlus {
+			cfgParams.HealthCheckEnabled = healthCheckEnabled
+		} else {
+			glog.Warning("Annotation 'nginx.com/health-checks' requires NGINX Plus")
+		}
 	}
 
-	if healthCheckMandatory, exists, err := GetMapKeyAsBool(ingEx.Ingress.Annotations, "nginx.com/health-checks-mandatory", ingEx.Ingress); exists {
-		if err != nil {
-			glog.Error(err)
+	if cfgParams.HealthCheckEnabled {
+		if healthCheckMandatory, exists, err := GetMapKeyAsBool(ingEx.Ingress.Annotations, "nginx.com/health-checks-mandatory", ingEx.Ingress); exists {
+			if err != nil {
+				glog.Error(err)
+			}
+			cfgParams.HealthCheckMandatory = healthCheckMandatory
 		}
-		cfgParams.HealthCheckMandatory = healthCheckMandatory
 	}
 
-	if healthCheckQueue, exists, err := GetMapKeyAsInt64(ingEx.Ingress.Annotations, "nginx.com/health-checks-mandatory-queue", ingEx.Ingress); exists {
-		if err != nil {
-			glog.Error(err)
+	if cfgParams.HealthCheckMandatory {
```

Comment:
I removed this section in the previous PR, but I think it's safest to put it back for now.